Aztec/Classes/Processor/HTMLProcessor.swift

```diff
@@ -66,8 +66,8 @@ open class HTMLProcessor: Processor {
     
     // MARK: - Initializers
     
-    public init(tag: String, replacer: @escaping Replacer) {
-        self.tag = tag
+    public init(for element: Element, replacer: @escaping Replacer) {
```

Comment:
A bit worried about this in the context of using HTMLProcessor in other places why force the use of an Element class? this could be just an convenience init no?